examples/jsm/loaders/LDrawLoader.d.ts

```diff
@@ -0,0 +1,21 @@
+import {
+	LoadingManager,
+	Group,
+	Material
+} from '../../../src/Three';
+
+export class LDrawLoader {
+	constructor(manager?: LoadingManager);
+	manager: LoadingManager;
+	path: string;
+
+	load(url: string, onLoad: (data: Group) => void, onProgress?: (event: ProgressEvent) => void, onError?: (event: ErrorEvent) => void) : void;
+	setPath(path: string) : this;
+	setFileMap(fileMap: Record<string, string>): void;
+	setMaterials(materials: Material[]): void;
+
+	parse(text: string, path?: string, onLoad: (data: Group) => void): void;
```

Comment:
The required parameter `onLoad` can not follow the optional parameter (`path`).

Reply:
Done.